spec/bookshelf-spec.ts

```diff
@@ -1491,11 +1491,12 @@ describe('Bookshelf relations', () => {
   });
 
   it('should give an option to sepcify relation types with an object', () => {
```

Comment:
Let's update the description of this test: 'should specify an option to format specific types using an object'